logging/log_visualization/log-visualization-ocp-console.adoc

```diff
@@ -0,0 +1,17 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+[id="log-visualization-ocp-console"]
+= Log visualization with the web console
+:context: log-visualization-ocp-console
+
+toc::[]
+
+You can use the {product-title} web console to visualize log data by configuring the {log-plug}.
+
+For information about configuring the plug-in during the {logging} installation, see xref:../../logging/cluster-logging-deploying.adoc#cluster-logging-deploy-console_cluster-logging-deploying[Installing the {logging} using the web console].
+
+If you have already installed the {logging} and want to retroactively configure the plug-in, use one of the following procedures.
```

Comment:
s/plugin/plug-in

Reply:
Minimalism nit: I think you could cut _retroactively_ and keep the intended meaning.